### PR TITLE
feat(chat): redesign of message template editor (Issue #2235)

### PR DESCRIPTION
**Pull request by IlyaBondar**
**Description:**

redesign of message template editor

Issues:

- Issue #2235 

**UI changes**

https://www.figma.com/design/lwL5jneveZczAj5RLcLFH2/DIAL-Chat?node-id=12326-85343&m=dev

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2253.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by irinakartun — DISMISSED** [review]

File: apps/chat/src/components/Chat/ChatMessage/ChatMessageTemplatesModal/ChatMessageTemplatesModal.tsx [hunk 1]
```diff
@@ -165,50 +187,45 @@ export const ChatMessageTemplatesModal = ({
...
+                  collapsed ? 'inline-block whitespace-nowrap' : 'mt-3 block',
+                )}
+              >
+                {showMore && (
+                  <button
+                    onClick={() => setCollapsed(!collapsed)}
+                    className="flex leading-5 text-accent-primary"
+                    data-qa={showMore ? 'show-less' : 'show-more'}
+                  >
+                    {!collapsed ? 'Show less' : 'Show more'}
```
Comment (Alexander-Kezik) [inline apps/chat/src/components/Chat/ChatMessage/ChatMessageTemplatesModal/ChatMessageTemplatesModal.tsx]: translation

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2253.nightly-test.deltixhub.io
>E2E tests status: success

**Review by nartovm — DISMISSED** [review]

**Review by Alexander-Kezik — DISMISSED** [review]

**Comment by IlyaBondar** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2253.nightly-test.deltixhub.io
>E2E tests status: success

**Review by Alexander-Kezik — APPROVED** [review]

**Review by nartovm — APPROVED** [review]

**Pull request merged**